domain/src/main/java/org/fao/geonet/domain/GeonetEntity.java

```diff
@@ -173,7 +173,7 @@ protected static boolean isGetter(Method method) {
      * @param descName
      * @return
      */
-    protected static boolean isNotLabel(final String descName) {
+    protected static boolean isLabel(final String descName) {
```

Comment:
I see a rename that change the semantics, but the method content looks the same. Previous name was not semantically correct?

Reply:
It was a double negation, so I simplified it as @cmangeat suggested to make it easier to read

Reply:
No, it wasn't semantically correct also.